.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,67 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+    paths:
+      - 'taqueria-plugin-*/docker/*'
+  create:
+    tags:
+      - 'v*'
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+
+  build-and-push-docker:
```

Comment:
You can remove the docker suffix since this job is already in the docker workflow

Reply:
That makes sense. Changed